platforms/cpuinfo.go

```diff
@@ -30,6 +32,9 @@ import (
 // Present the ARM instruction set architecture, eg: v7, v8
 var cpuVariant string
 
+//go:linkname runtimeGoArm runtime.goarm
```

Comment:
Does this linkage require a certain version of Go? I know the Windows ARM support was recently merged, but wasn't sure if this `runtime.goarm` predates these patches and has always been available when `GOARCH == arm`

Reply:
It seems to me that runtime.goarm has been in runtime/runtime2.go for at least two years.

@jordanrh1 in case he has some additional context.

Reply:
Correct, `runtime.goarm` has been around for a while.  However, `runtime.goarm` does not necessarily match the instruction set of the CPU. GOARM could be less than the version of the CPU.

Windows only supports ARMv7 for arm32 platforms, and ARMv8 for aarch64. So if the architecture is "arm", you know the architecture version is at least v7, and if the architecture is "arm64", you know the architecture version is at least v8. You could use the following logic to avoid `runtime.goarm`.
```
switch runtime.GOARCH {
default:
    variant = "unknown"
case "arm":
    variant = "v7"
case "arm64":
    variant = "v8"
}
```

Reply:
Updated based on recommendation of @jordanrh1 , our Go expert.

Reply:
Looks good.